src/features/leda-nft/services/item.service.ts

```diff
@@ -18,6 +18,11 @@ export default class ItemService extends HttpService implements IItemService {
     return data;
   }
 
+  async getNewest(qty: number): Promise<Item[]> {
+    const { data } = await this.instance.get(`${this.endpoint}/newest/${qty}`);
```

Comment:
I would make the endpoint to look like this: api/items/newest?qty=2. All the filtering criteria should be in the query string. Note: Make sure to have a max amount allowed in the backend, so the endpoint is not abusively used.